interfaces/apparmor/template.go

```diff
@@ -815,6 +814,16 @@ var privDropAndChownRules = `
   #capability kill,
 `
 
+// coreSnippet contains apparmor rules specific only for
+// snaps on native core systems.
+//
+var coreSnippet = `
```

Comment:
Isn't this what `defaultCoreRuntimeTemplateRules` is for?

Reply:
Ah good question, no the the `defaultCoreRuntimeTemplateRules` are apparmor rules for snaps with the `core` base, not for snaps running on a core native system

Reply:
OK, thanks for the explanation!